plugins/outputs/zabbix/README.md

```diff
@@ -0,0 +1,406 @@
+# Zabbix Output Plugin
+
+This plugin send metrics to [Zabbix](https://www.zabbix.com/) via
+[traps][traps].
+
+It has been tested with versions
+[3.0](https://www.zabbix.com/documentation/3.0/en/manual/appendix/items/trapper)
+,
+[4.0](https://www.zabbix.com/documentation/4.0/en/manual/appendix/items/trapper)
+and
+[6.0](https://www.zabbix.com/documentation/6.0/en/manual/appendix/items/trapper)
+.
+
+[traps]: https://www.zabbix.com/documentation/current/en/manual/appendix/items/trapper
+
+It should work with newer versions as long as Zabbix does not change the
+protocol.
+
+## Global configuration options <!-- @/docs/includes/plugin_config.md -->
+
+In addition to the plugin-specific configuration settings, plugins support
+additional global and plugin configuration settings. These settings are used to
+modify metrics, tags, and field or create aliases and configure ordering, etc.
+See the [CONFIGURATION.md][CONFIGURATION.md] for more details.
+
+[CONFIGURATION.md]: ../../../docs/CONFIGURATION.md#plugins
+
+## Configuration
+
+```toml @sample.conf
+# Send metrics to Zabbix
+[[outputs.zabbix]]
+  ## Address and (optional) port of the Zabbix server
+  address = "zabbix.example.com:10051"
+
+  ## Send metrics as type "Zabbix agent (active)"
+  # agent_active = false
+
+  ## Add prefix to all keys sent to Zabbix.
+  # prefix = "telegraf."
+
+  ## Name of the tag that contains the host name
+  # host_tag = "host"
```

Comment:
What is the behavior if a user is omitting this tag with the agent setting `omit_hostname = true`?

Reply:
https://github.com/influxdata/telegraf/pull/13739/files#diff-a148a78339729b36ca6334c99c1c6168e65a5b4369e6ed325a613dbbb9a9ddceR233-R240

Reply:
@adrianlzt - I'd like the fallback to be documented please.

Reply:
Added in 12e2d7e09